boards/pulga/include/board.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ * Copyright (C) 2022 Universidade de São Paulo
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf52840dk
+ * @{
+ *
+ * @file
+ * @brief       Board specific configuration for the Pulga board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Sebastian Meiling <s@mlng.net>
+ * @author      Geovane Fedrecheski <geonnave@gmail.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "board_common.h"
+#include "mtd.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    LED pin configuration
+ * @{
+ */
+/* FIXME: all LEDs are on by default. */
```

Comment:
You want to swap `LED0_OFF` and `LED0_ON` then

Reply:
In fact, I realized it is only inverted when `saul` is used. I updated the comment.